src/obr/OpenFOAM/case.py

```diff
@@ -229,14 +229,25 @@ def finished(self) -> bool:
             return self.latest_log.footer.completed
         return False
 
-    def fetch_latest_log(self) -> None:
-        solver = self.controlDict.get("application")
+    @property
+    def solver(self):
+        return self.controlDict.get("application")
 
+    def fetch_logs(self) -> list[Path]:
+        solver = self.solver
         root, _, files = next(os.walk(self.path))
-        log_files = [f for f in files if f.endswith(".log") and f.startswith(solver)]
+        log_files = [
+            Path(root) / f for f in files if f.endswith(".log") and f.startswith(solver)
+        ]
         log_files.sort()
+        return log_files
+
+    def fetch_latest_log(self) -> None:
+        log_files = self.fetch_logs()
         if log_files:
-            self.latest_log_path_ = Path(root) / log_files[-1]
+            self.latest_log_path_ = log_files[-1]
```

Comment:
I may be mistaken, but does `log_files.sort()` not sort the files lexicographically?

Reply:
Yes, but since we have a timestamp in the log file name sorting lexicographically also sorts by timestamp.